components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -198,6 +193,25 @@ func (backend *ESClient) setDailyLatestToFalse(ctx context.Context, nodeId strin
 	return err
 }
 
+func (backend *ESClient) UpdateProjectTags(ctx context.Context, projectTaggingRules map[string]*iam_v2.ProjectRules) ([]string, error) {
```

Comment:
This is the function that the common project update manager calls to start the update process.